### ISSUE TITLE
[a11y] Images must have alternate text

### ISSUE THREAD

**Issue by pdehaan** [mozilla/blurts-server#455]
via aXe.

```html
<img id="header-wave" class="wave" src="/img/wave.svg">

<img src="/img/subscribe-icon.svg">

<img class="logo" src="/img/firefox-logo.png">

<img id="footer-wave" class="wave" src="/img/wave.svg">
```

To solve this violation, you need to... Fix at least one (1) of these issues:
- Element does not have an `alt` attribute
- `aria-label` attribute does not exist or is empty
- `aria-labelledby` attribute does not exist, references elements that do not exist or references elements that are empty or not visible
- Element has no `title` attribute or the `title` attribute is empty
- Element's default semantics were not overridden with `role="presentation"`
- Element's default semantics were not overridden with `role="none"`

**Comment by verde79** [comment]
I'd like to fix this

**Comment by lesleyjanenorton** [comment]
Closing this issue since these images are no longer in use, but feel free to reopen if there are images without `alt`s that we've missed!
